integrations/segmentio/lib/index.js

```diff
@@ -316,11 +316,11 @@ Segment.prototype.normalize = function(message) {
   }
   if (this.options.addBundledMetadata) {
     var bundled = keys(this.analytics.Integrations);
-    var maybeBundledConfigIds = this.options.maybeBundledConfigIds
+    var maybeBundledConfigIds = this.options.maybeBundledConfigIds;
 
     // Generate a list of bundled config IDs using the intersection of
     // bundled destination names and maybe bundled config IDs.
-    var bundledConfigIds = []
+    var bundledIds = [];
```

Comment:
One theory I have is that `bundledConfigIds` was previously exposed in renderer snippet and may be stale and overriding the values here.

Reply:
https://github.com/segmentio/ajs-renderer/pull/256/files#diff-ca733fb7a822baffec24f4f426ccb07fce89f5b4d5856f848be4845d1d40e12bL61